transports/janus_websockets.c

```diff
@@ -1198,56 +1201,63 @@ static int janus_websockets_common_callback(
 				}
 
 				/* Check if we have a pending/partial write to complete first */
-				if(ws_client->buffer && ws_client->bufpending > 0 && ws_client->bufoffset > 0
-						&& !g_atomic_int_get(&ws_client->destroyed) && !g_atomic_int_get(&stopping)) {
```

Comment:
These atomic checks should not be removed.

Reply:
Added again the checks in the OR form to let the function early return.

Reply:
@lminiero Hi，May I asy why these atomic checks should not be removed ? 
It had checked before of this code branch, isn't ? 
What if ws_client->destroyed is not volicate int rather than int ? 
if so, Shoud it checked here ?